### PR TITLE
-fixed, text gets cut on widget

### PR DESCRIPTION
**Pull request by herou**

### CODE REVIEW

**Comment by herou** [comment]
issue #284

File: app/build.gradle [hunk 1]
```diff
@@ -3,9 +3,9 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
```
Comment (tibbi) [inline app/build.gradle]: this is so wrong, you should not change anything in this file. Just comment it out locally if needed.

Comment (herou) [reply app/build.gradle]: Not able to run the project without commenting  :/

Comment (tibbi) [reply app/build.gradle]: you can comment it on your pc, just dont submit the changes to the pull request

**Comment by herou** [comment]
Please take a look, I uncommented the lines of code in grandle.

**Comment by tibbi** [comment]
ye, thanks. I will test it later, Im not working on that app now.

**Comment by herou** [comment]
> ye, thanks. I will test it later, Im not working on that app now.

I can help you with the other app as well if you want.

**Comment by tibbi** [comment]
seems like it didnt break anything in my case, so if it fixed something for you, guess it is good to go. Thanks

**Comment by herou** [comment]
Hi, thank you.Took time.

**Comment by tibbi** [comment]
well ye, I wasnt working on the app and I dont want to jump around between them

**Comment by tibbi** [comment]
it actually broke widget scrolling, so Im reverting this whole thing

**Pull request merged**